Prose/ProseLib/Sources/SidebarFeature/Footer/Footer.swift

```diff
@@ -86,28 +144,49 @@ struct FooterState: Equatable {
   var avatar: FooterAvatarState
   var actionButton: FooterActionMenuState
 
+  @BindableState var sheet: Sheet?
+
   init(
     credentials: UserCredentials,
     teamName: String = "Crisp",
     statusIcon: Character = "🚀",
     statusMessage: String = "Building new features.",
     avatar: FooterAvatarState = .init(avatar: .placeholder),
-    actionButton: FooterActionMenuState = .init()
+    actionButton: FooterActionMenuState = .init(),
+    sheet: Sheet? = nil
```

Comment:
I'm not a big fan of these extensive initializers. Do we really need to inject the sheet property? Generally I think the initializers should only accept what they really need. Don't allow any misuse in production code. Everything else can be set via properties if we need to do so in tests. And these properties should be `internal` at most (which is already the case here).